src/triage/cli.py

```diff
@@ -186,6 +192,14 @@ def experiment(self):
         return experiment
 
     def __call__(self, args):
+
+        if args.setup is not None:
```

Comment:
it would be nice if there were some default filename that we looked for, *e.g.* `./experiment.py`.

I know y'all might not like that name ... but I kind of do :wink: 

Then, this condition here would just need to additionally check that the file exists, *e.g.*:

    if args.setup or os.path.exists('experiment.py'):
        setup_path = args.setup or 'experiment.py'
        logging.info('loading configuration from %s', setup_path)
        ...